app-admin/ruri/spec

```diff
@@ -0,0 +1,4 @@
+VER=3.8
+SRCS="git::commit=tags/v$VER;copy-repo=true::https://github.com/Moe-Hacker/ruri"
+CHKSUMS="SKIP"
+REL=1
```

Comment:
We do not start from REL=1, drop this line for first-time introduction/updates.

Reply:
Also, missing CHKUPDATE - see [documentation](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) (I assume you read Chinese as that was what you used in your last bug report). Anitya is preferred.

Reply:
> 另外，缺少 CHKUPDATE - 请参阅[文档](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md)（我假设您阅读中文，因为这是您在上次错误报告中使用的）。Anitya 是首选。



> Also, missing CHKUPDATE - see [documentation](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) (I assume you read Chinese as that was what you used in your last bug report). Anitya is preferred.

Obviously this isn't the one I'm looking for.
https://release-monitoring.org/project/82162/

So I tried using the github api.

Reply:
> > 另外，缺少 CHKUPDATE - 请参阅[文档](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md)（我假设您阅读中文，因为这是您在上次错误报告中使用的）。Anitya 是首选。
> 
> > Also, missing CHKUPDATE - see [documentation](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) (I assume you read Chinese as that was what you used in your last bug report). Anitya is preferred.
> 
> Obviously this isn't the one I'm looking for. https://release-monitoring.org/project/82162/
> 
> So I tried using the github api.

Just create an account and add your own - the GitHub API would require a personal access token/API key to use with `aosc-findupdate` and is undesirable.